src/domain/affiliate.js

```diff
@@ -107,7 +108,7 @@ class Affiliate {
             code: this.code,
             category: this.category,
             imageCredential: this.imageCredential,
-            plan: this.plan // this.plan ? JSON.parse(this.plan.toJson()) : this.plan,
+            plan: this.plan ? JSON.parse(this.plan.toJson()) : this.plan
```

Comment:
Esta condicion significa  que : 
Si existe un plan haceme un JSON.parse(UN JSON DE PLAN), si no dame ese mismo  plan ?

Reply:
`plan: this.plan && JSON.parse(this.plan.toJson())`